ecl/agentexec/agentexec.cpp

```diff
@@ -129,8 +115,16 @@ int CEclAgentExecutionServer::run()
 
     try 
     {
-        while (started)
+        while (true)
         {
+#ifdef _CONTAINERIZED
+            if (!pool->waitAvailable(10000))
+            {
+                  if (config->getPropInt("@traceLevel", 0) > 2)
```

Comment:
indented by 6 - and cause mismatching closing }

Reply:
Fixed